apps/src/templates/projects/PersonalProjectsPublishedCell.jsx

```diff
@@ -0,0 +1,65 @@
+import React, {Component, PropTypes} from 'react';
+import {connect} from 'react-redux';
+import FontAwesome from '../FontAwesome';
+import Button from '../Button';
+import i18n from '@cdo/locale';
+import {showPublishDialog} from './publishDialog/publishDialogRedux';
+import {unpublishProject} from './projectsRedux';
+
+class PersonalProjectsPublishedCell extends Component {
+  static propTypes = {
+    projectId: PropTypes.string.isRequired,
+    projectType: PropTypes.string.isRequired,
+    isPublished: PropTypes.bool.isRequired,
+    isPublishable: PropTypes.bool.isRequired,
+    publishMethod: PropTypes.oneOf(['button', 'chevron']).isRequired,
```

Comment:
these would be best as constants exported by this file:
```js
export const publishMethods = {
  PUBLISH: 'publish',
  CHEVRON: 'chevron',
}
```
and then referenced as `publishMethods.CHEVRON`, here and in PersonalProjectsTable.jsx .